### DISCUSSION TITLE
20231130 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by cgbosse** [nautilus-cyberneering/.github#73] [Meetings]
Time: 8:30 UTC (9:30 in Vaduz)

Location:
<https://meeting.hodl10.io/ConsistentClassroomsMergeSideways>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- CGarnham(@da2ce7)  - x
- JCelano(@josecelano) - x
- CBosse(@cgbosse) - x
- MMelchor(@mario-nt) - x
- GByrne(@grmbyrn) - x
- MVanDijke(@WarmBeer) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by mario-nt** [comment]
### Index

- https://github.com/torrust/torrust-index/pull/369

### GUI

- https://github.com/torrust/torrust-index-gui/pull/297

### Discussions
- https://github.com/torrust/torrust-tracker/pull/512
- https://github.com/torrust/torrust-index/pull/409

### Other

- Installed Docker

### Training/Research

- Typescript errors, exceptions, try/catch
- Best practices for handling errors/exceptions

**Comment by josecelano** [comment]
### Torrust Tracker

- https://github.com/torrust/torrust-tracker/issues/508
  - TODO: https://github.com/torrust/torrust-tracker/issues/524

### Torrust Index

- https://github.com/torrust/torrust-index/issues/408

### Torrust Index GUI

Overhaul Containerization:

- https://github.com/torrust/torrust-index-gui/issues/349
  - https://github.com/torrust/torrust-index-gui/issues/295
  - https://github.com/torrust/torrust-index-gui/issues/263
  - https://github.com/torrust/torrust-index-gui/issues/363
  - https://github.com/torrust/torrust-index-gui/issues/375
  - https://github.com/torrust/torrust-index-gui/issues/381
  - https://github.com/torrust/torrust-index-gui/issues/183

Others:

- https://github.com/torrust/torrust-index-gui/issues/379
- https://github.com/torrust/torrust-index-gui/issues/207
- https://github.com/torrust/torrust-index-gui/issues/368
- https://github.com/torrust/torrust-index-gui/issues/371
- All dependencies updated except:
  - Nuxt: https://github.com/torrust/torrust-index-gui/issues/393

Notes:

- No security alerts from Dependabot.
- Disabled Synk to check `Dockerfile` security. It does not work by default with `Containerfile`. I have to find out how to set up a custom container file.

### Torrust Website

- PR review: https://github.com/torrust/torrust-website/pull/108
- PRE review: https://github.com/torrust/torrust-website/pull/110
